src/extension/features/budget/show-available-after-savings/index.js

```diff
@@ -0,0 +1,65 @@
+import { Feature } from 'toolkit/extension/features/feature';
+import { isCurrentRouteBudgetPage } from 'toolkit/extension/utils/ynab';
+import { getEmberView } from 'toolkit/extension/utils/ember';
+import { addToolkitEmberHook } from 'toolkit/extension/utils/toolkit';
+import { createBudgetBreakdownElement } from 'toolkit/extension/features/budget/subtract-upcoming-from-available/index';
+
+export class ShowAvailableAfterSavings extends Feature {
+  shouldInvoke() {
+    return isCurrentRouteBudgetPage();
```

Comment:
we'll want to just return true here. it's not obviously but the `shouldInvoke(); -> invoke();` flow is called once YNAB is loaded. if you were to refresh the page on the accounts screen, we still want to attach your hook. so we should just make this always return true.

you can then remove your `onRouteChanged` at the bottom of the file